packages/templates/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "declarationDir": "dist-types"
+  },
+  "references": [
+    { "path": "../migration" },
+    { "path": "../react" }
+  ],
+  "include": [
+    "src/**/*.json",
```

Comment:
@swissspidy Let me know if we should import `.json` files in some other way. Currently, it looks like I have to add a separate line to include the files with `.json` extension to the config + add the extension to each import as well
(`import { default as template } from './template.json';` instead of
`import { default as template } from './template';`).

Reply:
Seems like that's how we have to do it 👍